pkg/plugin/step.go

```diff
@@ -22,6 +22,7 @@ const (
 
 var (
 	defaultResolution int64 = 1500
+	safeResolution    int64 = 11000
```

Comment:
what is `safeResolution` and why it is 11s?

Reply:
I have removed the safe interval. grafana used it because 
``` golang
// For safety, limit the number of returned points per timeseries.
	// This is sufficient for 60s resolution for a week or
    // 1h resolution for a year.
	if end.Sub(start)/step > 11000 {
		err := errors.New("exceeded maximum resolution of 11,000 points per timeseries. Try decreasing the query resolution (?step=XX)")
		return nil, &apiError{errorBadData, err}
```